js/repeater-list.js

```diff
@@ -655,6 +656,7 @@
 			$row.data('item_data', rows[index]);
 
 			$row.on('click.fu.repeaterList', function () {
+				if(self.isDisabled) return;
```

Comment:
this is technically correct, but as per our style guide we are supposed to write it as follows:
if (self.isDisabled) { return; }